### PR TITLE
Added description of Docs capability for "alerts"

### LINKED ISSUE #9094: [Docs] Add description of Docs capability for "alerts"
### Current State
Meshery Docs has a common include file "alert.html" which offers formatting for notes, warnings, and other informational callouts to readers. This feature isn't documented in the Docs Contributing guide, though.

### Desired State
Enhance https://docs.meshery.io/project/contributing/contributing-docs which a section describing this feature and include an example.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Debaditya-Som**
**Notes for Reviewers**

Changes Made:

- Added a new section in https://docs.meshery.io/project/contributing/contributing-docs that describes the "alerts.html" feature.
- Included a clear example of how to use the "alerts.html" feature in documentation.
- Ensured proper formatting and styling for clarity and consistency.

This PR fixes #9094 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0848e023b47c2bc05482d58e107633da7105f07f at: https://65379bfe063f8f156a1dbb37--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@coder12git would you like to review?

File: docs/v0.5/project/contributing/contributing-docs.html
Comment (leecalcote) [inline docs/v0.5/project/contributing/contributing-docs.html:1]: The v0.5 version of the docs is archived - a snapshot in time.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for putting forth effort here, @Debaditya-Som 👍

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow at 7:30 pm IST / 9:00 am CT,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by Debaditya-Som** [comment]
> Let's discuss this on the Meshery dev call tomorrow at 7:30 pm IST / 9:00 am CT, Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

Yes surely

**Comment by Debaditya-Som** [comment]
> Thanks for putting forth effort here, @Debaditya-Som 👍

You're welcome :)

**Comment by coder12git** [comment]
@Debaditya-Som Let me know, what help you need :)

**Comment by coder12git** [comment]
@Debaditya-Som actually you need to add a new section describing "alerts.html" in this file -`docs/pages/project/contributing/contributing-docs.md`, not the one you did

**Comment by Debaditya-Som** [comment]
> @Debaditya-Som actually you need to add a new section describing "alerts.html" in this file -`docs/pages/project/contributing/contributing-docs.md`, not the one you did , that version of the docs is archived now .

Okay got it, so I need to create a section in the mentioned file. Could you tell me whether the documentation that I have done in the archived version is accurate or not?

**Comment by leecalcote** [comment]
@coder12git ?

**Comment by coder12git** [comment]
> > @Debaditya-Som actually you need to add a new section describing "alerts.html" in this file -`docs/pages/project/contributing/contributing-docs.md`, not the one you did , that version of the docs is archived now .
> 
> Okay got it, so I need to create a section in the mentioned file. Could you tell me whether the documentation that I have done in the archived version is accurate or not?

@Debaditya-Som looks good to be, just move the given content to the specified file!

**Comment by Debaditya-Som** [comment]
> 
> @Debaditya-Som looks good to me, just move the given content to the specified file!

I have made the necessary changes, kindly review it.

**Comment by coder12git** [comment]
> > @Debaditya-Som looks good to me, just move the given content to the specified file!
> 
> @leecalcote ,@coder12git I have made the necessary changes, kindly review it.



> > @Debaditya-Som looks good to me, just move the given content to the specified file!
> 
> @leecalcote ,@coder12git I have made the necessary changes, kindly review it.

Also, do the changes suggested by me in the content and  you need to revert your commits too so that only those changes will be shown in which you have done changes, in your case just this file `docs/pages/project/contributing/contributing-docs.md`.

**Comment by Debaditya-Som** [comment]
> 
> Also, do the changes suggested by me in the content and you need to revert your commits too so that only those changes will be shown in which you have done changes, in your case just this file `docs/pages/project/contributing/contributing-docs.md`.

Yes, I have also made the changes that was suggested by you, only the updated commit, that is the one made in the specified file is made and the previous changes were reverted.

**Comment by coder12git** [comment]
> > Also, do the changes suggested by me in the content and you need to revert your commits too so that only those changes will be shown in which you have done changes, in your case just this file `docs/pages/project/contributing/contributing-docs.md`.
> 
> Yes, I have also made the changes that was suggested by you, only the updated commit, that is the one made in the specified file is made and the previous changes were reverted.

No, it'a not, first check carefully what changes i told you to make. Check the reviews that i have wrote in both files.
Also, by reverting i mean that you need to revert your PR, so that there is only commit made inside the specified file.

**Review by coder12git — CHANGES_REQUESTED** [review]

**Comment by coder12git** [comment]
> > Also, do the changes suggested by me in the content and you need to revert your commits too so that only those changes will be shown in which you have done changes, in your case just this file `docs/pages/project/contributing/contributing-docs.md`.
> 
> Yes, I have also made the changes that was suggested by you, only the updated commit, that is the one made in the specified file is made and the previous changes were reverted.

By revert, i mean revert your PR to only those commits which shows changes in the suggested file.

**Comment by Debaditya-Som** [comment]
> 
> By revert, i mean revert your PR to only those commits which shows changes in the suggested file.

image


I'm facing this error while reverting the initial commit

**Comment by Debaditya-Som** [comment]
I am facing merge conflict while working on this branch, I have created a new branch and submitted a new PR : #9226 , kindly review that PR and ignore this one

**Pull request closed without merging**